pip/cmdoptions.py

```diff
@@ -523,6 +524,47 @@ def only_binary():
 )
 
 
+def _good_hashes():
+    """Return names of hashlib algorithms at least as strong as sha256."""
+    # Remove getattr when 2.6 dies.
```

Comment:
I don't think 2.7 has this before like, 2.7.9 so probably we can't get rid of it until we get rid of 2.7.